tests/lambda_endpoints/test_queries.py

```diff
@@ -0,0 +1,708 @@
+"""Test queries lambda"""
+import datetime
+import json
+
+import pytest
+from sqlalchemy.orm import Session
+
+from sds_data_manager.lambda_code.SDSCode import queries
+from sds_data_manager.lambda_code.SDSCode.database import models
+
+
+@pytest.fixture()
+def setup_test_data(test_engine):
+    metadata_params = {
+        "file_path": "test/file/path/imap_hit_l0_raw_20251107_20251108_v02-01.pkts",
+        "instrument": "hit",
+        "data_level": "l0",
+        "descriptor": "raw",
+        "start_date": datetime.datetime.strptime("20251107", "%Y%m%d"),
+        "end_date": datetime.datetime.strptime("20251108", "%Y%m%d"),
+        "version": "v02-01",
+        "extension": "pkts",
+    }
+
+    # Add data to the file catalog
+    with Session(test_engine) as session:
+        session.add(models.FileCatalog(**metadata_params))
+        session.commit()
+
+    return ""
+
+
+@pytest.fixture()
+def expected_response():
+    expected_response = json.dumps(
+        str(
+            [
+                (
+                    1,
+                    "test/file/path/imap_hit_l0_raw_20251107_20251108_v02-01.pkts",
+                    "hit",
+                    "l0",
+                    "raw",
+                    datetime.datetime(2025, 11, 7, 0, 0),
+                    datetime.datetime(2025, 11, 8, 0, 0),
+                    "v02-01",
+                    "pkts",
+                    None,
+                )
+            ]
+        )
+    )
+    return expected_response
+
+
+def test_start_date_query(setup_test_data, test_engine, expected_response):
+    """Test that start date can be queried"""
+    event = {
```

Comment:
Can you reduce these down to the required values? I think that might just be the query parameters section and maybe a few others. But it would be a bit easier to parse what we are testing specifically.

Another way to combine things would be to put this in a `pytest.parametrize("event", [start_date_event, end_date_event, ...)` so you could eliminate a little boilerplate that way as well and have pytest do the looping for you with different queries.